features/standard/UserCreation.feature

```diff
@@ -0,0 +1,50 @@
+Feature: User management
+  As an administrator
+  In order to manage user to my site
```

Comment:
Low prio but it should read e.g. "to manage users on my site"

Reply:
> Always glad to see new scenarios coming!
> 
> Q: Will this be compatible with Commerce?

Yes, it will be. We will merge it up to 2.1 branch as well.